src/transaction.ts

```diff
@@ -93,6 +96,30 @@ export interface RunUpdateCallback {
   (err: null|ServiceError, rowCount: number): void;
 }
 
+export interface CommitRequest {
```

Comment:
Can't we just use `ICommitRequest`?

Reply:
Long story short: I wish, but not with how things are currently set up
Explanation: Generated files annotate the `transactionId` property with `?: Uint8Array|null`, but Transaction has an `id` property annotated as `string|UInt8Array` (we assign the `x.transactionId` value to `this.id` in transaction). Creating an interface that mirrors `ICommitRequest` but alters `transactionId` to include `string` seemed like the best way forward